css/style.css

```diff
@@ -70,13 +77,36 @@ body {
   display: flex;
   flex-direction: column;
 }
+.logo__bow {
+  height: 10px;
+  width: 19px;
+  margin: 12px 0px 0px;
+}
+@media (min-width: 1024px) {
+  .logo__bow {
+    height: 19px;
+    width: 37px;
+    margin: 16px 0px 0px;
+  }
+}
+.logo__name {
+  width: 117px;
+  height: 6.5px;
```

Comment:
You don't have to specify height of an image if you declared its height already (unless you want to change the aspect ratio but in this case I don't think you do).

Reply:
general comment - this is just our internal projects for learning purposes but once you start working on our real projects, try to always add code that's relevant to the given feature. What I mean is - this is f/hero branch that should include only work related to hero. The above changes concern the header though. Sometimes we can sneak some small changes to other components within our current branch but try to avoid it, with bigger PRs it may get confusing :)

Reply:
The logo loses is shape when I comment out the height in for the different breakpoint... I have just entered the height and width for each breakpoint based on the figma design - should I have done something differently so it changes in size more dynamically?

Reply:
as for the width/height: you can add both width and height but usually you don't have to - setting only a width will leave the height as auto so the image will scale proportionately. When I remove the height from .logo__bow or .logo__name (but leave the width), nothing changes (as expected).  We can have a call later today if you want to test it together.